VmAgent.Core/Interfaces/SessionHostContainerConfiguration.cs

```diff
@@ -26,6 +26,23 @@ public class SessionHostContainerConfiguration : SessionHostConfigurationBase
         {
             _dockerClient = dockerClient;
         }
+        /// <summary>
+        /// /
+        /// </summary>
+        /// <param name="assignmentId"></param>
+        /// <param name="instanceNumber"></param>
+        /// <returns></returns>
+        public SessionHostContainerConfiguration(
```

Comment:
why do we need the extra constructor? Can't we use the original one?

Reply:
ISessionHostManager has information on LinuxContainersOnWindows so I created an extra constructor to pass ISessionHostManager, or do you mean there is another way - like adding a new variable in SessionHostsStartInfo?

Reply:
apologies, I should have been more clear. I see there is an existing constructor (declared above this one), couldn't we add the ISessionHostManager argument to this one instead of creating an additional one? That might need way more changes in the code though

Reply:
I was not sure if it is okay to update the construct because other services might be using core libraries and calling the constructor.. you think it is okay?

Reply:
gotcha - since we don't need the ISessionHostManager anywhere in this class, can't we just add a boolean isRunningLinuxContainersOnWindows variable to the original constructor? With default value "false"?